src/external/ACSAdapter/AMSFileManager.ts

```diff
@@ -93,7 +121,8 @@ class AMSFileManager {
 
         try {
             const result = {
-                amsReferences: JSON.stringify(fileIds)
+                amsReferences: JSON.stringify(fileIds),
+                amsreferences: JSON.stringify(fileIds)

```

Comment:
is this for some sort of backward compatibility?

Reply:
That's correct